Makefile

```diff
@@ -83,5 +85,9 @@ publish:
 	fi
 	az storage blob upload-batch -d porter/mixins/$(MIXIN)/$(PERMALINK) -s $(BINDIR)/$(VERSION)
 
+install:
```

Comment:
Same comment as with the helm mixin, what do you think about doing a `mkdir -p` so that the azure mixin directory exists first before we copy?

Reply:
Good call; updated as done in https://github.com/deislabs/porter-helm/pull/10#discussion_r250320823